compatibility/BUILD.bazel

```diff
@@ -17,3 +18,98 @@ java_runtime_conformance(
     name = "java_conformance_v3.25.0",
     gencode_version = "3.25.0",
 )
+
+# Breaking change detection for well-known types and descriptor.proto.
+buf_breaking_test(
+    name = "any_proto_breaking",
+    targets = ["//:any_proto"],
+    config = ":buf.yaml",
+    against = "@com_google_protobuf_previous_release//:any_proto",
+)
+
+buf_breaking_test(
+    name = "api_proto_breaking",
+    targets = ["//:api_proto"],
+    config = ":buf.yaml",
+    against = "@com_google_protobuf_previous_release//:api_proto",
+)
+
+buf_breaking_test(
+    name = "descriptor_proto_breaking",
```

Comment:
We should add plugin.proto, cpp_features.proto, and java_features.proto too

Reply:
Thanks, should be added now.